mycore.org/content/de/documentation/frontend/frontend_xeditor.html

```diff
@@ -880,6 +880,32 @@ <h5>Validierungsregeln mit <code>&lt;xed:validate&gt;</code></h5>
 <p>validiert alle Datumsangaben, so dass sie entweder nur das Jahr, Jahr plus Monat oder Jahr, Monat und Tag in MODS W3CDTF Syntax enthalten.</p>
 </div>
 
+
+<div>
+    <h5>Kontextabhängige Validierung</h5>
+<p>In bestimmten Situationen kann es vorkommen, dass eine Validierungsregel ausgelassen werden soll.</p> 
+
+<p>Syntax:</p>
+{{< highlight xml "linenos=table">}} <xed:validate relevant-if="..." />{{< /highlight >}}
+
+<p>Das Attribut @relevant-if gibt in Form eines XPath-Tests an, unter welchen Bedingungen diese Validierungsregel geprüft werden soll. Hat das Attribut keinen Wert, wird
+die Validierungsregel immer geprüft.</p>
+
+<p>Beispiel: Die Validierungsregel, die prüft, dass ein übersetzter Titel einen Untertitel haben muss, soll nur dann überprüft werden, wenn der übersetzte Titel vorhanden ist und
+der Originaltitel einen Untertitel hat. Andernfalls soll die Validierung ausgelassen und das zugehörige Feld weder als gültig noch als ungültig markiert werden.</p>
+
+{{< highlight xml "linenos=table">}}<xed:bind xpath="mods:titleInfo[@type='translated']">
+  <xed:bind xpath="mods:title">
+    <input type="text" ... />
+  </xed:bind>
+  <xed:bind xpath="mods:subTitle">
+    <input type="text" ... />
+    <xed:validate required="true" relevant-if="../mods:title!='' and ../../mods:titleInfo[not(@type)]/mods:subTitle!=''" />
```

Comment:
Shouldn't it be enough to test if the node exists?
relevant-if="../mods:title  and ../../mods:titleInfo[not(@type)]/mods:subTitle"

Reply:
no, because the referenced `mods:title`- and `mods:subtitle`-elements always exists, whether they're empty or not, so the whole expression would always evaluate to `true`

Reply:
`relevant-if="../mods:title/text() and ../../mods:titleInfo[not(@type)]/mods:subTitle/text()"` would be another, but longer option